lib/armValidator/roundTripValidator.ts

```diff
@@ -106,22 +106,35 @@ export function diffRequestResponse(
   return rest;
 }
 
-export function buildLiveValidationIssue(errorCode: string, path: string): LiveValidationIssue {
+export function buildLiveValidationIssue(
+  errorCode: string,
+  path: string,
+  it: any
+): LiveValidationIssue {
   let severity, message;
   switch (errorCode) {
     case "ROUNDTRIP_INCONSISTENT_PROPERTY": {
+      const details = ` Value in request: "${it.oldValue}", Value in response: "${it.value}", path: "${path}".`;
       severity = roundTripValidationErrors.ROUNDTRIP_INCONSISTENT_PROPERTY.severity;
-      message = roundTripValidationErrors.ROUNDTRIP_INCONSISTENT_PROPERTY.message({});
+      message = roundTripValidationErrors.ROUNDTRIP_INCONSISTENT_PROPERTY.message({}) + details;
```

Comment:
what's the `{}` for? Can we put the value inside the message? like:
`The property's value 'westcentralus1' in the GET response is different from what was set 'westcentralus' in the preceding PUT request.`